### DISCUSSION TITLE
How do i assign alpha value to colors dynamically

### DISCUSSION THREAD

**Discussion by aurkaxi** [polybar/polybar#2952] [Q&A] [answered]
in my config.ini, i use 
```ini
[colors]
background = ${xrdb:color0}
```

i want it transparent in some cases. i tried 
```ini
[bar/main]
background = #00${colors.background: -6}
```
but it goes straight to solid black with zero transparency.

also why is the color format #AARRGGBB and not #RRGGBBAA

**Comment by patrick96** [comment, marked as answer]
Polybar's configuration doesn't have this kind of string or color manipulation features. You have to directly define the full colors in your xresources. If you want to do string manipulation, do it in a startup script that then exports an environment variable.


> also why is the color format #AARRGGBB and not #RRGGBBAA

I don't know if there was a particular reason for this, but it's been this way for 7 years, so no changing it now.

**Reply by aurkaxi** [reply]
Thank you sir.
